src/containers/Balance/Balance.tsx

```diff
@@ -129,7 +128,8 @@ class Balance extends React.Component<Props, State> {
           <label>Transfer amount</label>
           <div>
             <input
-              type="number"
+              type="text"
+              pattern="[0-9]*"
```

Comment:
Why the `*`?

Reply:
Means match any amount of the characters 0-9.